src/Creitive/Breadcrumbs/Breadcrumbs.php

```diff
@@ -350,6 +356,35 @@ public function setListElement($element)
 		$this->listElement = $element;
 	}
 
+/**
+     * Sets the icon breadcrumbs.
+     *
+     * @param string $classIcon
+     */
+    public function setIcon($classIcon)
+    {
+        if (!is_string($classIcon) && !is_null($classIcon))
+        {
+            throw new \InvalidArgumentException(
+                'Breadcrumbs::setIcon() only accepts strings or NULL, but '
+                . (is_object($classIcon) ? get_class($classIcon) : gettype($classIcon))
+                . ' given: ' . print_r($classIcon, true)
+            );
+        }
+
+        $this->icon = " <i class='{$classIcon}'></i> ";
```

Comment:
Also, what do you think about simply storing the actual passed icon in `$this->icon`? Then `setIcon()` and `getIcon()` would be symmetrical, whereas right now, `getIcon()` returns something notably different than what was passed into `setIcon()`.

Also, right now, if `null` is passed, `$this->icon` will be a string containing an empty `<i>` element with no class set.